ui/nuxeo-document-permissions/nuxeo-popup-permission.js

```diff
@@ -187,20 +187,20 @@ import '../widgets/nuxeo-user-suggestion.js';
           </paper-dialog-scrollable>
 
           <div class="buttons">
-            <paper-button dialog-dismiss>[[i18n('popupPermission.cancel')]]</paper-button>
+            <paper-button dialog-dismiss class="secondary">[[i18n('popupPermission.cancel')]]</paper-button>
             <dom-if if="{{!updatingACE}}">
               <template>
-                <paper-button noink class="primary" on-click="doCreateAndAdd" id="createAndAddPermissionButton">
+                <paper-button noink class="primary small" on-click="doCreateAndAdd" id="createAndAddPermissionButton">
```

Comment:
Same question about `small`.
Just want to understand the criteria in order to give better feedback.

Reply:
yes, for the permissions tab we discussed with @carolinagouveia to have the btns small. Not sure if there's a specific guideline, I think we're handling it case-by-case.